android/app/src/main/java/com/woowacourse/ody/presentation/meetings/MeetingsViewModel.kt

```diff
@@ -32,6 +37,23 @@ class MeetingsViewModel(
             }
         }
 
+    override fun navigateToEta(meetingId: Long) =
+        viewModelScope.launch {
```

Comment:
반환값이 Job인 이유가 있을까요? 이 반환값을 활용하고 있는 곳이 없는 것 같아서요. 
```suggestion
    override fun navigateToEta(meetingId: Long) {
        viewModelScope.launch {
```
이렇게 변경하면 되지 않을까요??